### PR TITLE
ADD command

### PR DESCRIPTION
**Pull request by Vadim222888**
add get brunch vadim

### CODE REVIEW

File: git/commands.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ After running the command respective git's technical files would be created.
 Before starting work, you need to make some settings:
 
 ```
+
 git config --global user.name "Your Name" # specify the name with which the commits will be signed
```
Comment (YuriiBaryshev) [inline git/commands.md:24]: Please use the common style of the document. If you are to state optional parameters use <parameter> notation

File: git/commands.md [hunk 2]
```diff
@@ -111,6 +113,7 @@ To create branch You should run
 
 ```
 git branch <name_branch>
+git branch vadim
```
Comment (YuriiBaryshev) [inline git/commands.md]: Please don't change this line. I suppose the students of younger courses wouldn't appreciate their branches all named `vadim`

File: git/commands.md [hunk 3]
```diff
@@ -34,6 +35,7 @@ git add -p       # show new/changed files one by one with their changes and a qu
 
 ##### Removing changes from the index
 ```
+
 git reset            # remove from the index all changes added to it (in the working directory all changes will be saved), the antipode of git add
```
Comment (YuriiBaryshev) [inline git/commands.md]: The description of the command is better to provide before the block of commands

**Review by YuriiBaryshev — CHANGES_REQUESTED** [review]

**Comment by Vadim222888** [comment]
command

**Pull request merged**